pylxd/models/container.py

```diff
@@ -16,4 +16,6 @@
 
 class Container(Instance):
 
```

Comment:
drop the extra blank line here and in the next hunk?

Reply:
good call, fixed